addons/mail/static/src/core/common/thread_model.js

```diff
@@ -554,6 +557,14 @@ export class Thread extends Record {
         }
         return this._store.Message.get(Math.max(...previousMessages.map((m) => m.id)));
     }
+
+    onUpdateSuggestedRecipients() {
+        this.suggestedRecipients
+            .filter((s) => s.checked === undefined)
+            .map((s) => {
+                s.checked = true;
+            });
```

Comment:
Could be single line.

Also `map` is meant to transform/`return` a value from a list, for side effect with no return it is better to use `forEach`.

But I don't like `forEach` because it makes the stack and debugging more complicated due to callback functions.

A simple `for ... of` loop would be just as clear and efficient and probably take even less lines.